src/backend/access/table/table.c

```diff
@@ -199,7 +199,7 @@ CdbTryOpenTable(Oid relid, LOCKMODE reqmode, bool *lockUpgraded)
 {
```

Comment:
Need more commit messages in current commit. Because current PR won't be squash when merged.

Reply:
I will try my best ) Added small description about Cloudberry features which should be taken into account.